AppController/djinn.rb

```diff
@@ -3551,10 +3551,19 @@ def prime_database()
     retries = 10
     loop {
       return if system("#{prime_script} --replication #{replication}")
-      Djinn.log_warn("Failed to prime database. #{retries} retries left.")
-      Kernel.sleep(SMALL_WAIT)
       retries -= 1
+      Djinn.log_warn("Failed to prime database. #{retries} retries left.")
+
+      # Run a repair and try to prime one more time.
+      if retries == 1
+        @state = 'Running a Cassandra repair.'
+        Djinn.log_warn(@state)
+        system("#{NODETOOL} repair")
```

Comment:
Can we add a comment with a link and short description to the Cassandra docs for this?